design/cdk-bootstrap.md

```diff
@@ -352,6 +337,21 @@ Here is the JSON of the bootstrap CloudFormation template:
       "Description": "List of the ManagedPolicy ARN(s) to attach to the CloudFormation deployment role",
       "Default": "",
       "Type": "CommaDelimitedList"
+    },
+    "FileAssetsBucketName": {
+      "Description": "A user-provided custom name to use for the file assets bucket",
+      "Default": "",
+      "Type": "String"
+    },
+    "CustomKmsKeyId": {
+      "Description": "A user-provided ID of a custom KMS key used for encrypting the file assets",
+      "Default": "",
+      "Type": "String"
+    },
+    "CustomContainerAssetsRepositoryName": {
```

Comment:
Do we need the "Custom" prefix?

Reply:
Yes, the "Custom" prefix has since been removed. Let me rebase this PR on top of the latest master (in master, this parameter is [simply called `ContainerAssetsRepositoryName`](https://github.com/aws/aws-cdk/blob/52fd078abcc93876aa7423949e1e6090644cf95d/packages/aws-cdk/lib/api/bootstrap/bootstrap-template.json#L24-L24)).